backend/pkg/helm/handler.go

```diff
@@ -178,3 +111,61 @@ func (r *restConfigGetter) ToRESTMapper() (meta.RESTMapper, error) {
 
 	return expander, nil
 }
+
+type stat struct {
+	Status string
+	Err    error
+}
+
+// getStatus returns the status of the release.
+func (h *Handler) getStatus(actionName, releaseName string) (*stat, error) {
```

Comment:
Could this be `getReleaseStatus`? Reading it I didn't know what status it was getting.